packages/framework/src/Component/Elasticsearch/IndexRepository.php

```diff
@@ -181,7 +182,7 @@ protected function findIndexNamesForAlias(string $aliasName): array
 
         $indexesWithAlias = array_keys($indexes->getAlias(['name' => $aliasName]));
         if (count($indexesWithAlias) === 0) {
-            throw ElasticsearchIndexException::noIndexFoundForAlias($aliasName);
+            throw new ElasticsearchNoIndexForAlias($aliasName);
```

Comment:
Hmmm, you have created new Exception here, but you do not use it in catch. What was the reason for this refactoring? Maybe as mentioned before - would not it be better to create Exception class for all exceptions in ElasticsearchIndeException?

Reply:
Yeah, it should not be here as the design has changed.